shared/css/conversation.css

```diff
@@ -376,6 +376,34 @@ html[dir="rtl"] .settings-menu.dropdown-menu {
   }
 }
 
+/* Stream paused */
+.focus-stream > .room-inner-info-area > .remote-stream-paused {
+  background-color: #fff;
+  border: solid 1px #a6a6a6;
+  box-shadow: 0 2px 4px rgba(0,0,0,.25);
+  display: inline-block;
+  margin: auto;
+  padding: 10px 15px;
+}
+
+.focus-stream > .room-inner-info-area > .remote-stream-paused > h1 {
+  font-size: 16px;
+  line-height: 30px;
+  padding-left: 42px;
```

Comment:
When you're using pixel values for font-related properties, instead of em/ rem, can you add a comment above them to explain why? My guess is that we can use em/ rem values here as well...